vector-app/build.gradle

```diff
@@ -0,0 +1,554 @@
+
+    buildTypes {
+        debug {
+            applicationIdSuffix ".debug"
+            resValue "string", "app_name", "Element dbg"
+
+            buildConfigField "boolean", "LOW_PRIVACY_LOG_ENABLE", "false"
+            // Set to true if you want to enable strict mode in debug
+            buildConfigField "boolean", "ENABLE_STRICT_MODE_LOGS", "false"
+
+            signingConfig signingConfigs.debug
+
+            if (project.hasProperty("coverage")) {
+                testCoverageEnabled = coverage.enableTestCoverage
+            }
+        }
+
+        release {
+            resValue "string", "app_name", "Element"
+
+            buildConfigField "boolean", "LOW_PRIVACY_LOG_ENABLE", "false"
+            buildConfigField "boolean", "ENABLE_STRICT_MODE_LOGS", "false"
+
+            postprocessing {
+                removeUnusedCode true
+                removeUnusedResources true
+                // We do not activate obfuscation as it makes it hard then to read crash reports, and it's a bit useless on an open source project :)
+                obfuscate false
+                optimizeCode true
+                proguardFiles 'proguard-rules.pro'
+            }
+            // signingConfig signingConfigs.release
+        }
+    }
+
+    flavorDimensions "store"
+
+    productFlavors {
+        gplay {
+            apply plugin: 'com.google.gms.google-services'
+            afterEvaluate {
+                tasks.matching { it.name.contains("GoogleServices") && !it.name.contains("Gplay") }*.enabled = false
+            }
+
+            dimension "store"
+            isDefault = true
+            versionName "${versionMajor}.${versionMinor}.${versionPatch}${getGplayVersionSuffix()}"
+        }
+
+        fdroid {
+            dimension "store"
+            versionName "${versionMajor}.${versionMinor}.${versionPatch}${getFdroidVersionSuffix()}"
+        }
+    }
+
+    lintOptions {
+        lintConfig file("lint.xml")
+
+        checkDependencies true
+        abortOnError true
+    }
+
+    compileOptions {
+        sourceCompatibility versions.sourceCompat
+        targetCompatibility versions.targetCompat
+    }
+
+    kotlinOptions {
+        jvmTarget = "11"
+        freeCompilerArgs += [
+                "-opt-in=kotlin.RequiresOptIn",
+                // Fixes false positive "This is an internal Mavericks API. It is not intended for external use."
+                // of MvRx `by viewModel()` calls. Maybe due to the inlining of code... This is a temporary fix...
+                "-opt-in=com.airbnb.mvrx.InternalMavericksApi",
+                // Opt in for kotlinx.coroutines.FlowPreview too
+                "-opt-in=kotlinx.coroutines.FlowPreview",
+                // Opt in for kotlinx.coroutines.ExperimentalCoroutinesApi too
+                "-opt-in=kotlinx.coroutines.ExperimentalCoroutinesApi",
+        ]
+    }
+
+    buildFeatures {
+        viewBinding true
+    }
+}
+
+dependencies {
+    implementation project(':vector')
```

Comment:
When I first read the PR, I thought the dependency would be the opposite. So maybe in the file `settings.xml` put this module above `:vector`.